lib/ledger-node-operations.js

```diff
@@ -23,11 +51,11 @@ module.exports = class LedgerNodeOperations {
    * operation - the operation to perform.
    * options - a set of options used when adding the operation.
```

Comment:
There are no options yet.

Reply:
Is that true? We currently destructure the first parameter so that we get the `operation`. The first parameter we are destructuring could very well be `options`, right?

Reply:
Eh?  That sounds confusing.  I'm noting that this looks like an artifact from our older style of commonly having a final "options" argument which isn't there in this case.